go/tasks/pluginmachinery/catalog/async_client_impl.go

```diff
@@ -14,11 +14,14 @@ import (
 	"github.com/flyteorg/flytestdlib/errors"
 
 	"github.com/flyteorg/flyteplugins/go/tasks/pluginmachinery/workqueue"
+
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/core"
 )
 
 const specialEncoderKey = "abcdefghijklmnopqrstuvwxyz123456"
 
 var base32Encoder = base32.NewEncoding(specialEncoderKey).WithPadding(base32.NoPadding)
+var emptyLiteralMap = core.LiteralMap{Literals: map[string]*core.Literal{}}
```

Comment:
should this be defined in the `hashing.go` file instead? I believe it's the only place it's being used right?

Reply:
Thanks for spotting - moved it over 👍 30ca1642ce7fe834781ebad7916bed5d90a84ecf